### PR TITLE
26.2 final changes

### PR DESCRIPTION
**Pull request by glozow**
bins were uploaded 2 weeks ago on June 18
website PR: https://github.com/bitcoin-core/bitcoincore.org/pull/1039

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -2,7 +2,7 @@ AC_PREREQ([2.69])
 define(_CLIENT_VERSION_MAJOR, 26)
 define(_CLIENT_VERSION_MINOR, 2)
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_RC, 1)
+define(_CLIENT_VERSION_RC, 0)
 define(_CLIENT_VERSION_IS_RELEASE, true)
 define(_COPYRIGHT_YEAR, 2023)
```
Comment (stickies-v) [inline configure.ac:7]: nit (which also affects manpages)
```suggestion
define(_COPYRIGHT_YEAR, 2024)
```

**Review by stickies-v — APPROVED** [review]
ACK 41985528b9c7faf19a05b00ecaec9b8f2d1b3895

I rebased this branch on the latest 26.x branch (6bbfdcfb6d6a620216eb8cdb50c238113cd77e6e, includes https://github.com/bitcoin/bitcoin/pull/30319) but manpages are unchanged, so this should be fine.

File: configure.ac (hunk 1 above)
Comment (glozow) [reply configure.ac]: thanks, taken

**Comment by glozow** [comment]
Rebased and made copyright year update, thanks @stickies-v!

**Review by stickies-v — APPROVED** [review]
ACK eef5dbc21b3fd52069f2f0855fb76a13234ebbf3

**Review by fanquake — APPROVED** [review]
ACK eef5dbc21b3fd52069f2f0855fb76a13234ebbf3 If you want, you could also backport the changes to get the ASAN job running again, but that isn't blocking here.

**Review by theStack — APPROVED** [review]
ACK eef5dbc21b3fd52069f2f0855fb76a13234ebbf3

**Review by Bbjj88h — APPROVED** [review]

**Review by Bbjj88h — APPROVED** [review]

**Review by Shuhaib07 — COMMENTED** [review]
Approve

**Review by Thompson1985 — APPROVED** [review]

**Review by bainpa — COMMENTED** [review]
MAZENTERPRISE

**Pull request merged**